src/Language/Fixpoint/Types/Names.hs

```diff
@@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
   , dropSym
   , headSym
   , lengthSym
+  , mappendSym
```

Comment:
No Niki please don't do that, the separator is there for a reason. what is
the problem with inserting the separator?

On Monday, May 16, 2016, Niki Vazou <notifications@github.com> wrote:

> In src/Language/Fixpoint/Types/Names.hs
> <https://github.com/ucsd-progsys/liquid-fixpoint/pull/227#discussion_r63376213>
> :
>
> > @@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
> >    , dropSym
> >    , headSym
> >    , lengthSym
> > +  , mappendSym
>
> @ranjitjhala <https://github.com/ranjitjhala> I cannot because
> symbolSyffix inserts a separator. I created another function in Names that
> does what I want, and do nor expose mappendSym, if this makes any difference
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ucsd-progsys/liquid-fixpoint/pull/227/files/019c3bdb49e2988b8a59fbff30cd84b7450b862b..1a6a2b4f034faee428e2bf1c6c08dcbfc1238b9c#r63376213>
>